### ISSUE TITLE
Some nodes with no edges are not being grouped with others.

### ISSUE THREAD

**Issue by HaroldTexas** [Microsoft/automatic-graph-layout#367]
I'm using the MDS layout.  Some but not all nodes with zero edges are being aligned to the right of the graph.  Is this an known issue?
![NodesWithZeroEdges](https://github.com/microsoft/automatic-graph-layout/assets/42121615/7778bd89-89d4-4355-9d28-746f20a3c5e3)

**Comment by levnach** [comment]
No,it is not a known issue. It is really not clear where to put the isolated nodes. For other nodes their neighbors help in positioning. The only constraint for the disjoint nodes is to avoid the overlaps. Do you have a suggestion?

**Comment by HaroldTexas** [comment]
If I leave off all the nodes that do not have an edge the bounding box is much smaller and the box sides are close to the nodes (small margin).  So, there appears to be logic to increase the size of the bounding box so the edgeless nodes can be placed along the side.  I'm thinking that logic could use the largest width of edgeless nodes (the height would be adjusted to keep the golden ratio).  This would allow all the edgeless nodes to fit along the side.  However, the sheer number of those nodes could run into a height issue.  So, the height may have to be increased to handle the number of nodes.  Although, there appears to be current logic that combines smaller nodes on the same line to prevent so many lines.  Definitely not a simple problem.

**Comment by levnach** [comment]
Please try the following : shorten the labels of the isolated nodes in the left top corner. You can modify them as "First string\\nSecond string". It will improve the layout I believe.

**Comment by HaroldTexas** [comment]
The layout is not always the same.  Sometimes the nodes in the top left are shorter then ones along the right edge.  I changed the layout packing method to columns.  This prevented edgeless nodes from being placed on the same line.  I'm not sure why the packing algorithm is creating 2 packages.  There appears to be plenty of room along the right edge.

![image](https://github.com/microsoft/automatic-graph-layout/assets/42121615/d01ca2cb-69b6-4ce7-90d0-ffdd591942f2)
